standard/unsafe-code.md

```diff
@@ -704,6 +704,7 @@ It is an error to use a captured local variable ([§11.17.6.2](expressions.md#11
 - The token “`&`” followed by a *variable_reference* ([§9.5](variables.md#95-variable-references)) to a moveable variable ([§22.4](unsafe-code.md#224-fixed-and-moveable-variables)) of an unmanaged type `T`, provided the type `T*` is implicitly convertible to the pointer type given in the `fixed` statement. In this case, the initializer computes the address of the given variable, and the variable is guaranteed to remain at a fixed address for the duration of the fixed statement.
 - An expression of an *array_type* with elements of an unmanaged type `T`, provided the type `T*` is implicitly convertible to the pointer type given in the fixed statement. In this case, the initializer computes the address of the first element in the array, and the entire array is guaranteed to remain at a fixed address for the duration of the `fixed` statement. If the array expression is `null` or if the array has zero elements, the initializer computes an address equal to zero.
 - An expression of type `string`, provided the type `char*` is implicitly convertible to the pointer type given in the `fixed` statement. In this case, the initializer computes the address of the first character in the string, and the entire string is guaranteed to remain at a fixed address for the duration of the `fixed` statement. The behavior of the `fixed` statement is implementation-defined if the string expression is `null`.
+- An expression of type other than *array_type* or `string`, provided the type there exists an accessible method or accessible extension method named `GetPinnableReference`, that method has no parameters, that method returns a `ref` or `ref readonly` variable of an *unmanaged_type* `T`, and `T*` is implicitly convertible to the pointer type given in the `fixed` statement. The `GetPinnableReference`. In this case, the initializer computes the address of the returned variable, and that variable is guaranteed to remain at a fixed address for the duration of the `fixed` statement. A `GetPinnableReference()` method can be used by the `fixed` statement when overload resolution (§11.6.4) produces exactly one function member and that function member satisfies the preceding conditions. The `GetPinnableReference` method should return a reference to an address equal to zero, such as that returned from `System.Runtime.CompilerServices.Unsafe.NullRef<T>()` when there is no data to pin.
```

Comment:
Looks like this has been mangled:
- "provided the type there exists an accessible method"
- "The `GetPinnableReference`."

Include the signature `ref [readonly] T GetPinnableReference()` as in the proposal rather than just describe it?

Reply:
Fixed.